### PR TITLE
feat[UI/Level]: options menu update and level 02 rework

### PR DESCRIPTION
**Pull request by Vlukkee**
### UI - Changes
Add Feedback on options menu to saved changes.
Now on options menu cant leave pressing start or esc.
Add new UI images.
Add UI changes on all levels.
Fix some UI img that dont fit with the template.

### Level Changes
Fix Level 02 Windows corridor.
Rework level 02 Armory. - Still need add rigids and fix IBL and lights-

I made the reworks in other map _LEVEL02_ALFA_V02

### CODE REVIEW

File: Game/Scripts/UIGameManager.cpp [hunk 1]
```diff
@@ -89,16 +89,29 @@ void UIGameManager::Update(float deltaTime)
...
+	{
+		if (optionMenuActive)
+		{
+			return;
+		}
+		else
+		{
+			OpenInGameMenu(!inGameMenuActive);
+		}
+	}
```
Comment (markmeliaa) [inline Game/Scripts/UIGameManager.cpp:112]:
```suggestion
	if ((input->GetKey(SDL_SCANCODE_ESCAPE) == KeyState::DOWN || 
	input->GetKey(SDL_SCANCODE_E) == KeyState::DOWN)
	 && inGameMenuActive && !optionMenuActive)
	{
	     OpenInGameMenu(!inGameMenuActive);
	}
```

Comment (markmeliaa) [reply Game/Scripts/UIGameManager.cpp:112]: `optionMenuActive` can be added to the `if statement` above, thus avoiding additional branching.

**Review by markmeliaa — COMMENTED** [review]
Great job Ale!! Just a one comment.

File: Game/Scripts/UIGameManager.cpp (hunk 1 above)
Comment (Vlukkee) [reply Game/Scripts/UIGameManager.cpp]: Done! Thanks!

**Review by AuxPort66 — APPROVED** [review]

**Review by markmeliaa — APPROVED** [review]

**Pull request merged**